extensions/big-data-cluster/src/bigDataCluster/controller/apiGenerated.ts

```diff
@@ -203,7 +204,7 @@ export class Dashboards {
     }
 }
 
-export class EndpointModel {
+export class EndpointModel implements bdc.IEndpointModel {
     'name'?: string;
```

Comment:
hmm, why are we quoting the property names in this class?

Reply:
This is generated from a tool that does this. Not something I wanted to bother going through and fixing (and would be annoying if we needed to re-generate the file)